src/items/FilePdf.tsx

```diff
@@ -41,11 +41,13 @@ const FilePdf: FC<FilePdfProps> = ({
     }
   };
 
+  const src = `https://pdfviewer.dev.graasp.org/pdf/web/viewer.html?file=${url}`;
+
   return (
     <embed
```

Comment:
I'm curious why this is not integrated as an iframe? We would also get security benefits if we tighten the cookie policy or simply serve from another domain.